pkgs/development/python-modules/python-on-whales/default.nix

```diff
@@ -37,12 +38,11 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "python_on_whales" ];
 
-  meta = with lib; {
+  meta = {
     description = "Docker client for Python, designed to be fun and intuitive";
     homepage = "https://github.com/gabrieldemarmiesse/python-on-whales";
-    changelog = "https://github.com/gabrieldemarmiesse/python-on-whales/releases/tag/${src.tag}";
-    license = licenses.mit;
-    maintainers = with maintainers; [ bcdarwin ];
-    mainProgram = "python-on-whales";
```

Comment:
No more `mainProgram` ?

Reply:
Okay

Reply:
Ah, thanks, somehow I wasn't sure if it was mandatory for this case, but just checked [https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes](the manual) and it is.